jasmine/src/game.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+class Game {
+  constructor(runningScore, bowlCard, scoreCard, index) {
+    this.runningScore = 0;
+    this.bowlCard = [];
+    this.scoreCard = [];
```

Comment:
There's a concept in programming called "single source of truth" which is where you avoid duplicating state wherever possible. This has a number of benefits, as it prevents any chance of the two variables getting out of sync, which ultimately reduces the chance of bugs. It also makes it easier to add functionality to our code. For example, if we wanted to add the functionality to edit a previous roll that had been entered incorrectly, currently we would need to update three different variables in different ways, whereas if the source of truth was just the rolls, and everything else was calculated from those each time, we would only need to change that piece of data, and we could be confident that the class would treat it like it had always been that way.

In the case of this class, you have the rolls, frame scores and runningScore stored separately, when the last two are derived from the first. Can you think of a way to simplify this so that you only need to store the rolls, and the scores can be calculated on the fly as needed?

The same goes for the index. As far as I can see, the index refers to how many frames have so far been entered. Could you use this, instead of storing it in its own instance variable?